src/fastoad/notebooks/02_explanation/03_scipy.ipynb

```diff
@@ -0,0 +1,388 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    # We can now define the bounds of the optimization. Essentially there are two constraints : both value must
</pre><p>define the bound of the design variables of the optimization problem.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Corrected in a6b9f26f